codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketServerProtocolHandshakeHandler.java

```diff
@@ -110,6 +105,10 @@ public void operationComplete(ChannelFuture future) throws Exception {
         }
     }
 
+    private boolean isNotWebSocketPath(FullHttpRequest req) {
+        return checkStartsWith ? !req.uri().startsWith(websocketPath) : !req.uri().equals(websocketPath);
```

Comment:
Do you think it worth it? This is just handshake that happens once on establishing connection. The goal here was to make code cleaner and easier to read, while proposed change is opposite.